### ISSUE TITLE
Blank Screen on result tab after latest update | postgresql

### ISSUE THREAD

**Issue by h4ckt1v1st** [mtxr/vscode-sqltools#648]
I am using sqltools with postgres db. Everything was working fine until latest update (0.22.10).
Now, whenever I run a query, it just show me blank screen. I can not see the loading spinner.

![image](https://user-images.githubusercontent.com/22280578/87149721-64e4f700-c2ce-11ea-96a4-31ad0bf93b4f.png)

Checked the SQLTools output. No error there.

````
['ls:info'] SQLTools Server started!

Using node runtime?: no
ExecPath: C:\Users\4013876\AppData\Local\Programs\Microsoft VS Code\Code.exe 

['ext:lc:info'] LanguageClient ready
namespaces set to  *,-babel*,-*:debug,-*:*:debug,-*:*:*:debug,-*:*:*:*:debug,-*:*:*:*:*:debug
['ls:info'] request to register plugin: "c:\Users\4013876\.vscode\extensions\mtxr.sqltools-driver-pg-0.1.0\out\ls\plugin.js"
['ls:ls-context:register-driver'] Driver postgresql registered!
['ls:ls-context:register-driver'] Driver aws redshift registered!
['ext:main:info'] EXECUTING => sqltools.executeQuery
['ext:main:info'] EXECUTING => sqltools.getConnections
['ext:main:info'] EXECUTING => sqltools.getConnections
['ls:cache:query-results:info'] DELETE keys starting with [KPI-Prod DB|PostgreSQL|172:21:17:57|fwdevops_kpis_db]...
['ls:cache:query-results:info'] DELETE keys starting with [KPI-Prod DB|PostgreSQL|172:21:17:57|fwdevops_kpis_db] completed!
['ls:cache:manager:info'] DELETE keys starting with [KPI-Prod DB|PostgreSQL|172:21:17:57|fwdevops_kpis_db]...
['ls:cache:manager:info'] DELETE keys starting with [KPI-Prod DB|PostgreSQL|172:21:17:57|fwdevops_kpis_db] completed!
['ext:main:info'] EXECUTING => sqltools.executeQuery
['ext:main:info'] EXECUTING => sqltools.getConnections
````

I tried rolling back to version 0.22.9 and it is working perfectly now.

**Comment by pgensler** [comment]
This happened to me as well today, I'm wondering if it's not finding the proper image file locations per the Developer Tools:

![image](https://user-images.githubusercontent.com/7308115/87234855-b0fd6d80-c39a-11ea-872c-486d4b1c53c5.png)

I'm on Windows 10 with VSCode 1.47.0
Extension: 0.22.10
![image](https://user-images.githubusercontent.com/7308115/87234846-9fb46100-c39a-11ea-89c1-7d6ce6fe3ea8.png)

**Comment by pgensler** [comment]
FYI @h4ckt1v1st Reverting the version of SQLTools to 0.22.19 fixes this for me via this issue: https://github.com/mtxr/vscode-sqltools/issues/639

**Comment by h4ckt1v1st** [comment]
> FYI @h4ckt1v1st Reverting the version of SQLTools to 0.22.19 fixes this for me via this issue: #639

Did you mean 0.22.9?
Yes, I tried that as well. I mentioned this at the very last of my isuue.

**Comment by mtxr** [comment]
Dup #639

**Comment by mtxr** [comment]
Fixed on v0.22.11.
